jsx/Form.js

```diff
@@ -909,15 +915,21 @@ class TextboxElement extends Component {
       elementClass = 'row form-group has-error';
     }
 
-    return (
-      <div className={elementClass}>
+    let label = null;
+    if (this.props.label) {
+      label = (
         <label className="col-sm-3 control-label" htmlFor={this.props.id}>
           {this.props.label}
           {requiredHTML}
         </label>
-        <div className="col-sm-9">
+      );
+    }
+    return (
+      <div className={elementClass}>
+        {label}
+        <div className={this.props.class}>
           <input
-            type="text"
+            type={this.props.type}
```

Comment:
@zaliqarosli when the type=password.

Reply:
could we maybe add a boolean isPassword prop that changes the type to password if true, and otherwise, text

Reply:
orr would be cool to have a separate password input react component

Reply:
@zaliqarosli also I made the default be "text" and I don't really think it's bad to have this option. I don't think we should design the react component in this case to where it restricts the developer from modifying the html element value.

Reply:
true! as long as the default is text, then i think we're fine

Reply:
Just my two cents, but I would think that if the input field is taking on an entirely new function it should probably deserve it's on component called `PasswordElement`. Perhaps Passwords and Textboxes are close enough that that isn't necessary though. Just something to consider. 

I also agree that if we are to keep passwords as `TexboxElement`, thhen a boolean prop of `password` would be better than passing any type you want to the `TextboxElement`

Reply:
@HenriRabalais @zaliqarosli I think making a new component for a password input is unnecessary and kind of ridiculous if I'm playing devil's advocate. I'll try to express why I think components should be designed without restricting the developer from accessing or changing the original html values in simple cases such as type=password. We shouldn't add properties with "new names" that need to be found & learned by developers and to do things of common knowledge in the html world. Most developers with html knowledge know how to change an input type to not display the text when it's a password being inputed.
I'm only writing this because I think the components right now are restricting access to classes and it shouldn't be done this way.

Reply:
Given the current structure of `Form.js` I don't think creating an entirely new component seems odd. Actually, it seems to me that a different input types are the distinguishing characteristic of defining a new component (select, button, date, etc.) in `Form.js`. I don't know everything that inputs of `type=password` are capable of, but if there are many options for them they could be expanded upon within a `PasswordElement` component.

The concern specifically with the change to `type={this.props.type}` is that someone could easily pass the value 'date' as a type prop. This shouldn't be allowed. There's a DateElement which provides other options specific to dates that should be used. A password prop flag would be able to handle this password configuration is a clearer and more controlled way. 

I may be wrong, but I think restricting access to classes and html functionality is a bit of the point. By doing this all forms across loris will be standardised. If components become more lenient, there will be less internal coherence within loris. I know that can be frustrating (I've been struggling a lot with Form.js because of Biobank), but I think the solution is to introduce new features (e.g. inline form, label-less form) rather than making everything customisable.

Reply:
@HenriRabalais Idk I just find it limiting and too much of encapsulation. I agree a developer could go and make a date input but I don't necessarily think this PR with the password is comparable to that situation. Now if the password input was to be heavily stylized with it's own feature of frontend verification for matching LORIS password rules.. then I would find an individual password component as a valid reason for existence because the logic would be associated with it and inside the component.
The classes changes were actually needed to preserve the Login style of LORIS. Please look at the login page and even test the PR. The indentation with labels looked awful on the Login Page, before I added the modifications to not require labels and which shouldn't break the other forms. I'm not sure if you're suggesting we have a new component for without label? I'm alright if that is to be done but I think the changes I made aren't bad.